spk/gitea/src/config.ini

```diff
@@ -0,0 +1,8 @@
+[repository]
+ROOT = ${SHARE_PATH}/gitea-repositories
+[server]
+SSH_DOMAIN = ${IP:=localhost}
+DOMAIN = ${IP:=localhost}
+ROOT_URL = http://${IP:=localhost}:${SERVICE_PORT}/
+[lfs]
+PATH = ${SHARE_PATH}/lfs
```

Comment:
You cannot use the shell variables here. 
You need some place-holders (like `@shared_folder@`) that you can replace using the `sed` command in service-setup.sh.

You find example code in spk/sabnzbd/src/service-setup.sh, spk/znc/src/service-setup.sh and others.